### ISSUE TITLE
User Guide Review

### ISSUE THREAD

**Issue by Rachelwrq** [OpenLiberty/draft-guide-rest-client-angular#20]
**Time: 20 minutes**

#### Defining a service to fetch data
- [x] `ArtistsService` shouldn't included the annotation above for hotspot
- [x] Under the first arrow, `ArtistsService` class and `AppComponent` class are hotspoted differently, should be the same for consistency. `AppComponent` should probably highlight the entire class
- [x] Under the first arrow, fourth paragraph, the first sentence, in the end "and returning the result." Probably shouldn't use present continuous tense.
- [x] People might not be familiar with what a `Promise` is if they are not familiar with Angular. Some explanation or links could be helpful

#### Defining a component to consume a service
- [x] One of the `AppComponent` does not have hotspot.

### User Testing Checklist
- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear to understand
- [x] Ensure no logic errors 
- [x] Resolve anything that does not work
- [x] Resolve anything usability issues (confusing or hard to follow)
- [x] Achieve basic understanding/working knowledge of the topic in the guide

**Comment by realModusOperandi** [comment]
Why shouldn't the `@Injectable` annotation be included in the hotspot for the `ArtistsService` class? It applies to the class and I think it should be included when highlighting the entire class. I'll open a PR to make the rest of the changes

**Comment by evelinec** [comment]
Thank you @realModusOperandi. It seems like we don't include the annotations in a hotspot. That has been the cases for other guides. Thus the user review had added the feedback.

**Comment by realModusOperandi** [comment]
Ok thanks, that explanation would be helpful whenever it comes up. I'll make the adjustment.

**Comment by evelinec** [comment]
- [ ] For the `https://promisejs.org` link you added, can you add it as `https://promisejs.org[^]` so it can be clicked to open in a new tab? Thanks.

Original sentence: 
A Promise is how JavaScript represents the state of an asynchronous operation. If you want to learn more, check out https://promisejs.org for an introduction.

- [ ] The `AppComponent` class in this sentence, "The AppComponent class implements the OnInit interface", should have a hotspot for the entire class, to be consistent with the other places where this class has a hotspot.

**Comment by evelinec** [comment]
Fixed the above 2 minor feedback.

**Comment by evelinec** [comment]
@Rachelwrq would you verify the changes for your feedback? If everything is good, close the issue. Thanks.

**Comment by evelinec** [comment]
Checked with @Rachelwrq who confirmed the changes are good. Closed.
